cnitool/cnitool.go

```diff
@@ -59,7 +60,6 @@ func parseArgs(args string) ([][2]string, error) {
 func main() {
 	if len(os.Args) < 4 {
 		usage()
-		return
```

Comment:
@dcbw thanks for quick response! this will never be called since `usage` calls `os.Exit(1)`? do you want me to remove this change cz it does not align the commit description? just checking on what you mean, thanks!

Reply:
@dcbw friendly ping.

Reply:
Nah, that's fine.